frontend/src/components/TaskList.jsx

```
@@ -0,0 +1,214 @@
+          tasks.map((task) => (
+            <div
+              key={task._id}
+              className={`${styles.taskItem} ${task.status === "Completed" ? styles.completed : ""}`}
+              onClick={() => openEditTaskModal(task)}
+              onKeyDown={(e) => {
+                if (e.key === 'Enter' || e.key === ' ') {
+                  openEditTaskModal(task);
+                }
+              }}
+              tabIndex={0}
+              role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)